.travis.yml

```diff
@@ -15,8 +15,10 @@ node_js:
     # Current status of node versions: https://github.com/nodejs/LTS/
     # We don't work with node 6 because it doesn't support package-lock
     # files which we need to avoid the broken version of base-x
```

Comment:
Maybe remove the historical explanation above? Our supported set is now effectively "LTS or newer", so I don't think we need to say much more to explain it. (Whatever seems best to you though!)

Reply:
I have to admit: I didn't actually read any of this. Will remove the explanation